src/index.js

```diff
@@ -17,9 +17,11 @@ const replacement = '--REDACTED--';
  * Module exports.
  */
 
-module.exports = (whitelist = []) => {
-  const terms = whitelist.join('|');
-  const paths = new RegExp(`^(${terms.replace('.', '\\.').replace(/\*/g, '.*')})$`, 'i');
+module.exports = (whitelist = [], blacklist = []) => {
```

Comment:
PR looks really good. The only thing that pops up to me is the arguments here. I would much prefer a named object like:

```
const anonymize = anonymizer({
  blacklist: ['secret'],
  whitelist: ['foo.*']
});
```

The only problem is that it would be a breaking-change. @fixe any thoughts?

Reply:
I will make this change